hacks/install_etcdctl

```diff
@@ -21,27 +21,36 @@ function show_help () {
 }
 
 function install () {
-  local version=$1
+  local version="${1:-v3.4.30}" # default version "v3.4.3
   local download_url
+  local arch
+  local platform
+  local pkg
+  local dest
 
-  if [ -z "$version" ]
-  then # fetch v3.4.26
-    version="v3.4.26"
+  if [[ $(whoami) == "root" ]]; then
+    dest="/usr/local/bin/etcdctl"
   else
-    if [[ ! $version == v* ]]
-    then
-      version="v${version}"
-    fi
+    dest="${DOTFILES_HOME}/bin/etcdctl"
   fi
+  arch="$(arch)"
```

Comment:
If I run `arch` in the container, it returns `x86_64` and hence the download url is wrong